extension_templates/classification.py

```diff
@@ -273,6 +274,14 @@ def get_test_params(cls, parameter_set="default"):
         # A parameter dictionary must be returned *for all values* of parameter_set,
         #   i.e., "parameter_set not available" errors should never be raised.
         #
+        # A good parameter set should primarily satisfy two criteria,
+        #   1. Chosen set of parameters should reduce the testing time.
+        #       This is vital for the case of Deep Classifiers where default values
```

Comment:
this sentence is too specific to DL classifiers, and also true for all other classifiers, so I would leave it out. It is vital in general

Reply:
You're right! Changed it to be type/class-agnostic via `This is vital for the cases where default values result in "big" models which..`

Reply:
Was waiting to clarify other points, updated.